src/templates/toolkit/bet/static.html.tt

```diff
@@ -169,7 +169,7 @@
                                 <label for="stop_loss" id="stop_loss_label"></label>
                             </div>
                             <div class="big-col">
-                                <input id="stop_loss" type="text" value="10" pattern="^([1-9]|[1-9][0-9]|[1-9][0-9][0-9]|[1-9][0-9][0-9][0-9]|[1-9][0-9][0-9][0-9][0-9]|[1][0][0][0][0][0])$" title="100,000 maximum"/><label></label>
```

Comment:
please check with quants whether stop_loss and stop_profit can be decimal, i think it can be